codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: off
+coverage:
+  status:
+    project:
+      default:
+        target: auto
```

Comment:
@benpatterson for now do we want auto? option: "auto" (must increase from parent commit or pull request base)
or should we set it to something that we know we are above (e.g. 85%)
http://docs.codecov.io/docs/commit-status